stable/redmine/README.md

```diff
@@ -183,9 +212,9 @@ See the [Parameters](#parameters) section to configure the PVC or to disable per
 The following example includes two PVCs, one for Redmine and another for MariaDB.
 
 1. Create the PersistentVolume
-1. Create the PersistentVolumeClaim
-1. Create the directory, on a worker
-1. Install the chart
+2. Create the PersistentVolumeClaim
+3. Create the directory, on a worker
+4. Install the chart
```

Comment:
This is actually the syntax to created a ordered list in gh markdown https://guides.github.com/features/mastering-markdown/. Not sure why it is.

Reply:
I am sorry. Just my mistake.
I hope I fixed it.